README.md

```diff
@@ -4,29 +4,13 @@ The Celeritas project implements HEP detector physics on GPU accelerator
 hardware with the ultimate goal of supporting the massive computational
 requirements of LHC-HL upgrade.
 
-# Installation and development
+# Installation
 
-This project requires external dependencies to build with full functionality.
-However, any combination of these requirements can be omitted to enable
-limited development on personal machines with fewer available components.
-
-- [CUDA](https://developer.nvidia.com/cuda-toolkit): on-device computation
```

Comment:
Did you move this list somewhere else?

Reply:
Yeah, it's in a table in `doc/infrastructure.rst`. It seems like too much detail for the top-level readme, plus it's a lot of duplication...

Reply:
Yep, once is enough.  Since it was already in `doc/infrastructure.rst` I missed it when searching the diffs :)